horace_core/sqw/PixelData/@PixelDataMemory/PixelDataMemory.m

```diff
@@ -114,6 +114,20 @@
     % File handling/migration. Does nothing on membased except dump_data
     % which recets raw data with input.
     methods
+        function obj = deactivate(obj)
+            % close all open file handles to allow file movements to new
+            % file/new location.
+            % Does nothing on memory based
+        end
+        function [obj,varargout] = activate(obj,varargin)
```

Comment:
Why use varargout here if only one is ever returned? Give it a name to match the FileBacked version

Reply:
if you give it a name, it will check it to be always set.  I never use output in PixelDataMemory